wordpress-mobile/WordPress-iOS
# Issues/5136 sharing icons

Fixes #5136 

To test: 
View the blog details for a wpcom site. 
View sharing settings. 
View sharing buttons.
Confirm that the icons for Google, WhatsApp, Skype, and Telegram are properly displayed and not using the default share icon. 

Needs review: @kurzee
